modules/microshift-get-nonrunning-cluster-id-kubesystem.adoc

```diff
@@ -4,13 +4,13 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-get-nonrunning-cluster-id-kubesystem_{context}"]
-= Getting the cluster ID of a nonrunning cluster
+= Getting the cluster ID of a stopped cluster
 
-For a cluster that is not running, you can get the cluster ID from the `cluster-id` file found in `/var/lib/microshift` directory.
+For a cluster that is not running, you can get the cluster ID from the `cluster-id` file found in the `/var/lib/microshift` directory.
 
 .Procedure
 
-. Get the cluster ID of the cluster that is not running by entering the following command:
+. Get the cluster ID of a cluster that is not running by entering the following command:
```

Comment:
Should we use the same sentence here as in https://github.com/openshift/openshift-docs/pull/74538/files#diff-90a06578ffc9ac91b414884800c6eeef3bd52dcf364a77bfca2aa144843944eeR26?

Reply:
Yes, we should.